### PR TITLE
Adding quick fix documenation

### PR DESCRIPTION
**Pull request by codepo8**
Adding documentation for the new release of the extension published 9/8

### CODE REVIEW

File: microsoft-edge/visual-studio-code/microsoft-edge-devtools-extension.md [hunk 1]
```diff
@@ -73,6 +73,23 @@ Issues are evaluated live while you edit your code.  As you type, you get feedba
 
 ![A possible issue being explained on an output element](microsoft-edge-devtools-extension-images/live-issue-reporting.png)
```
Comment (mikehoffms) [inline microsoft-edge/visual-studio-code/microsoft-edge-devtools-extension.md:75]: I didn't see tagging errors.  To fix the doc build, I'd try merging "main" into the PR.

File: microsoft-edge/visual-studio-code/microsoft-edge-devtools-extension.md [hunk 2]
```diff
@@ -73,6 +73,23 @@ Issues are evaluated live while you edit your code.  As you type, you get feedba
...
 
+#### Automated fixes and issue filtering
+
+When you hover over the element with the issue, you get a lightbulb icon indicating that there are quick fixes available.
+
+![An anchor element with a protocol specific href attribute highliighted as a problem with a lightbulb icon above it](microsoft-edge-devtools-extension-images/light-bulb.png)
```
Comment (mikehoffms) [inline microsoft-edge/visual-studio-code/microsoft-edge-devtools-extension.md:80]: @JasonAndrewWriter @codepo8 @Reezaali - This .png shows "example.com", might be a requirement to change it to "contoso.com" and change demo code to that.  Next .png too.

File: microsoft-edge/visual-studio-code/microsoft-edge-devtools-extension.md [hunk 3]
```diff
@@ -73,6 +73,23 @@ Issues are evaluated live while you edit your code.  As you type, you get feedba
...
+
+Activating the lightbulb icon shows a list of options. For example, if you added a link with a protocol relative URL, you get:
+
+![The quick fix panel opened next to the anchor element with the error offering the different options explained in the following text](microsoft-edge-devtools-extension-images/quick-fix-options.png)
```
Comment (mikehoffms) [inline microsoft-edge/visual-studio-code/microsoft-edge-devtools-extension.md:84]: This 2nd .png shows "example.com", might be a requirement to change it to "contoso.com" and change demo code to that.

File: microsoft-edge/visual-studio-code/microsoft-edge-devtools-extension.md [hunk 4]
```diff
@@ -73,6 +73,23 @@ Issues are evaluated live while you edit your code.  As you type, you get feedba
 
 ![A possible issue being explained on an output element](microsoft-edge-devtools-extension-images/live-issue-reporting.png)
 
+#### Automated fixes and issue filtering
```
Comment (mikehoffms) [inline microsoft-edge/visual-studio-code/microsoft-edge-devtools-extension.md:76]:
```suggestion
#### Automated Quick Fixes
```
Is "issue filtering" covered in this h4 section?  What's the relation between Quick Fix UI & issue filtering?

Comment (mikehoffms) [reply microsoft-edge/visual-studio-code/microsoft-edge-devtools-extension.md:76]: My PR clarifies this.

File: microsoft-edge/visual-studio-code/microsoft-edge-devtools-extension.md (hunk 3 above)
Comment (mikehoffms) [inline microsoft-edge/visual-studio-code/microsoft-edge-devtools-extension.md:84]:
```suggestion
![The quick fix panel opened next to the anchor element with the error offering the different options explained in the following text](microsoft-edge-devtools-extension-images/quick-fix-options-light.png)
```

Comment (mikehoffms) [reply microsoft-edge/visual-studio-code/microsoft-edge-devtools-extension.md:84]: I don't have a button to Resolve conversation, and I don't want to Delete this comment which was substantive.
@Reezaali can you Resolve the present conversation?

File: microsoft-edge/visual-studio-code/microsoft-edge-devtools-extension.md (hunk 2 above)
Comment (mikehoffms) [reply microsoft-edge/visual-studio-code/microsoft-edge-devtools-extension.md:80]: I don't have a button to Resolve conversation, and I don't want to Delete this comment which was substantive.
Pls Resolve the present conversation.

File: microsoft-edge/visual-studio-code/microsoft-edge-devtools-extension.md [hunk 5]
```diff
@@ -74,6 +74,27 @@ Issues are evaluated live while you edit your code.  As you type, you get feedba
...
+
+Using Quick Fixes, you can customize the error reporting of the extension to the needs of the current project.
+
+When you hover over an element that has an issue, you get a lightbulb icon indicating that there are Quick Fixes available:
+
+![An anchor element with a protocol-specific href attribute highlighted as a problem, by a wavy underline and lightbulb icon above it](microsoft-edge-devtools-extension-images/light-bulb.png)
+
+Clicking the lightbulb icon shows a list of options. For example, if you added a link with a protocol-relative URL, you get the following list of Quick Fixes to choose from:
+
+![The Quick Fix panel opened next to the anchor element with the error, offering several Quick Fix options](microsoft-edge-devtools-extension-images/quick-view-options.png)
```
Comment (mikehoffms) [inline microsoft-edge/visual-studio-code/microsoft-edge-devtools-extension.md:87]:
```suggestion
![The Quick Fix panel opened next to the anchor element with the error, offering several Quick Fix options](microsoft-edge-devtools-extension-images/quick-fix-options.png)
```
An inconsistency in previous commits re: naming shows that we should correct this.

File: microsoft-edge/visual-studio-code/microsoft-edge-devtools-extension.md [hunk 6]
```diff
@@ -74,6 +74,27 @@ Issues are evaluated live while you edit your code.  As you type, you get feedba
...
+#### Automated Quick Fixes and issue filtering
+
+Using Quick Fixes, you can customize the error reporting of the extension to the needs of the current project.
+
+When you hover over an element that has an issue, you get a lightbulb icon indicating that there are Quick Fixes available:
+
+![An anchor element with a protocol-specific href attribute highlighted as a problem, by a wavy underline and lightbulb icon above it](microsoft-edge-devtools-extension-images/light-bulb.png)
+
+Clicking the lightbulb icon shows a list of options. For example, if you added a link with a protocol-relative URL, you get the following list of Quick Fixes to choose from:
+
```
Comment (mikehoffms) [inline microsoft-edge/visual-studio-code/microsoft-edge-devtools-extension.md:86]: rename the png from quick-view-options.png to quick-fix-options.png

File: microsoft-edge/visual-studio-code/microsoft-edge-devtools-extension.md [hunk 7]
```diff
@@ -74,6 +74,27 @@ Issues are evaluated live while you edit your code.  As you type, you get feedba
...
 
 
+#### Automated Quick Fixes and issue filtering
+
+Using Quick Fixes, you can customize the error reporting of the extension to the needs of the current project.
+
+When you hover over an element that has an issue, you get a lightbulb icon indicating that there are Quick Fixes available:
+
+![An anchor element with a protocol-specific href attribute highlighted as a problem, by a wavy underline and lightbulb icon above it](microsoft-edge-devtools-extension-images/light-bulb.png)
+
```
Comment (mikehoffms) [inline microsoft-edge/visual-studio-code/microsoft-edge-devtools-extension.md:84]: delete the file light-bulb-light.png

File: microsoft-edge/visual-studio-code/microsoft-edge-devtools-extension.md [hunk 8]
```diff
@@ -74,6 +74,27 @@ Issues are evaluated live while you edit your code.  As you type, you get feedba
...
+Clicking the lightbulb icon shows a list of options. For example, if you added a link with a protocol-relative URL, you get the following list of Quick Fixes to choose from:
+
+![The Quick Fix panel opened next to the anchor element with the error, offering several Quick Fix options](microsoft-edge-devtools-extension-images/quick-view-options.png)
+
+You can select whichever Quick Fix you want to use to resolve the issue or stop reporting it as an issue:
```
Comment (mikehoffms) [inline microsoft-edge/visual-studio-code/microsoft-edge-devtools-extension.md:89]: delete the file quick-fix-options-light.png

File: microsoft-edge/visual-studio-code/microsoft-edge-devtools-extension.md (hunk 1 above)
Comment (captainbrosset) [reply microsoft-edge/visual-studio-code/microsoft-edge-devtools-extension.md:75]: Seems like the build worked fine this time.

**Review by captainbrosset — APPROVED** [review]
LGTM with Michael's comments addressed.

File: microsoft-edge/visual-studio-code/microsoft-edge-devtools-extension.md (hunk 7 above)
Comment (mikehoffms) [reply microsoft-edge/visual-studio-code/microsoft-edge-devtools-extension.md:84]: "delete the file light-bulb-light.png" - done locally in new forked branch user/mikehoffms/exten-img/cln

File: microsoft-edge/visual-studio-code/microsoft-edge-devtools-extension.md (hunk 6 above)
Comment (mikehoffms) [reply microsoft-edge/visual-studio-code/microsoft-edge-devtools-extension.md:86]: "rename the png from quick-view-options.png to quick-fix-options.png" - done locally in new forked branch user/mikehoffms/exten-img/cln

File: microsoft-edge/visual-studio-code/microsoft-edge-devtools-extension.md (hunk 5 above)
Comment (mikehoffms) [reply microsoft-edge/visual-studio-code/microsoft-edge-devtools-extension.md:87]: in .md, change from quick-view-options.png to quick-fix-options.png - done locally in new forked branch user/mikehoffms/exten-img/cln

File: microsoft-edge/visual-studio-code/microsoft-edge-devtools-extension.md (hunk 8 above)
Comment (mikehoffms) [reply microsoft-edge/visual-studio-code/microsoft-edge-devtools-extension.md:89]: "delete the file quick-fix-options-light.png" - done locally in new forked branch user/mikehoffms/exten-img/cln

**Pull request merged**